cafebabel/templates/base.html

```diff
@@ -3,6 +3,21 @@
 <meta name=viewport content="width=device-width, minimum-scale=1, initial-scale=1, shrink-to-fit=no">
 <title>{% block meta_title %}pan-European news, features and opinions{% endblock %} – Cafebabel</title>
 <meta name=description content="{% block meta_description %}{% endblock %}">
+<!--Social-->
+<meta property="og:url" content="{{ request.url }}">
+<meta property="og:title" content="{{ self.meta_title() }}">
+<meta property="og:image" content="{% block meta_image_url %}{% endblock %}"> <!-- to discuss -->
+<meta property="og:type" content="{% block meta_type %}website{% endblock %}">
+<meta property="og:description" content="{{ self.meta_description() }}">
+<meta property="og:site_name" content="Cafébabel">
+<meta property="og:locale" content="{% block meta_language %}{% endblock %}">
```

Comment:
Not sure if this will be manage on a per-block basis or per-website version.
We should probably detect the language based on the domain (fr: cafebabel.fr, en: cafebabel.co.uk, …).

Reply:
Let's keep it like it is right now. Seems as logical as detect the language on the domain 👍